src/pretalx/orga/templates/orga/settings/room_list.html

```diff
@@ -4,7 +4,9 @@
 {% block settings_content %}
 
 <div>
-    <legend>{{ request.event.rooms.count }} {% trans "Rooms" %}</legend>
+    <legend>{{ request.event.rooms.count }}
+        {% blocktrans count count=request.event.rooms.count %}Room{% plural %}Rooms{% endblocktrans %}
```

Comment:
`blocktrans` takes a `trimmed` argument that strips whitespace. You'd use it like `{% blocktrans trimmed count … %}`